test/accessibility/apps/discover.ts

```diff
@@ -46,105 +39,84 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
-      await PageObjects.discover.inputSavedSearchTitle('filterSearch');
-      await PageObjects.discover.clickConfirmSavedSearch();
-      await PageObjects.discover.openLoadSavedSearchPanel();
-      await PageObjects.discover.loadSavedSearch('filterSearch');
-      await a11y.testAppSnapshot();
-    });
-
-    it('click share button', async () => {
+    it('a11y test on share panel', async () => {
       await PageObjects.share.clickShareTopNavButton();
       await a11y.testAppSnapshot();
     });
 
-    it('Open sidebar filter', async () => {
+    it('a11y test on open sidenav filter', async () => {
       await PageObjects.discover.openSidebarFieldFilter();
       await a11y.testAppSnapshot();
-    });
-
-    it('Close sidebar filter', async () => {
       await PageObjects.discover.closeSidebarFieldFilter();
-      await a11y.testAppSnapshot();
     });
 
-    it('Add a field from sidebar', async () => {
+    it('a11y test on tables with columns view', async () => {
       for (const columnName of TEST_COLUMN_NAMES) {
-        await PageObjects.discover.clickFieldListItemAdd(columnName);
+        await PageObjects.discover.clickFieldListItemToggle(columnName);
       }
       await a11y.testAppSnapshot();
     });
 
-    it('Add more fields from sidebar', async () => {
-      for (const [columnName, value] of TEST_FILTER_COLUMN_NAMES) {
-        await PageObjects.discover.clickFieldListItem(columnName);
-        await PageObjects.discover.clickFieldListPlusFilter(columnName, value);
-      }
+    it('a11y test on save queries popover', async () => {
+      await PageObjects.discover.clickSavedQueriesPopOver();
       await a11y.testAppSnapshot();
     });
 
-    // Context view test
-    it('should open context view on a doc', async () => {
-      await retry.try(async () => {
-        await docTable.clickRowToggle();
-        // click the open action
-        const rowActions = await docTable.getRowActions();
-        if (!rowActions.length) {
-          throw new Error('row actions empty, trying again');
-        }
-        await rowActions[0].click();
-      });
+    it('a11y test on save queries panel', async () => {
+      await PageObjects.discover.clickCurrentSavedQuery();
       await a11y.testAppSnapshot();
     });
 
-    // Adding rest of the tests after https://github.com/elastic/kibana/issues/53888 is resolved
+    it('a11y test on toggle include filters option on saved queries panel', async () => {
+      await PageObjects.discover.setSaveQueryFormTitle('test');
+      await PageObjects.discover.toggleIncludeFilters();
+      await a11y.testAppSnapshot();
+      await PageObjects.discover.saveCurrentSavedQuery();
+    });
+
+    it('a11y test on saved queries list panel', async () => {
+      await PageObjects.discover.clickSavedQueriesPopOver();
+      await testSubjects.moveMouseTo(
+        'saved-query-list-item load-saved-query-test-button saved-query-list-item-selected saved-query-list-item-selected'
+      );
```

Comment:
What's this? Hover actions aren't accessible...

Reply:
I did something and now my reply isn't visible. Ok so this step is moving the mouse over on a saved queries list panel so delete icon is visible next to the name. And then I am testing the listing panel for a11y. Thanks!